### PR TITLE
Add Ryan's aboutus and PPP

### PR DESCRIPTION
**Pull request by ryanwalterlee**

### CODE REVIEW

**Comment by benmurphyy** [comment]
@ryanwalterlee need to add your part of the user guide too

File: docs/AboutUs.md [hunk 1]
```diff
@@ -40,10 +40,10 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Jean Doe
```
Comment (yokester98) [inline docs/AboutUs.md:41]: Need to change this to your full name

**Review by yokester98 — CHANGES_REQUESTED** [review]
Need to change that one portion in AboutUs to your full name!

**Comment by ryanwalterlee** [comment]
The user guide part, I only did the image, which is already there. I however changed the other image to suit our app. Let me know if there's any part I missed.

**Review by benmurphyy — CHANGES_REQUESTED** [review]
remove the bin folder items

**Review by benmurphyy — CHANGES_REQUESTED** [review]
one more part in user guide to change
Screenshot 2022-02-28 at 1 13 51 PM

**Review by benmurphyy — CHANGES_REQUESTED** [review]
for help part, the url for our user guide is
https://ay2122s2-cs2103-w16-1.github.io/tp/UserGuide.html

**Review by benmurphyy — APPROVED** [review]

**Pull request merged**